seqr/fixtures/1kg_project.json

```diff
@@ -382,6 +382,17 @@
         "success_story": "Differential treatement",
         "success_story_types": ["A", "D"]
     }
+},
+    {
+    "model": "seqr.familyanalysedby",
+    "pk": 1,
+    "fields": {
+        "guid": "FAB000001_1",
```

Comment:
Should we explicitly test the entry deleted while deleting the family?

Reply:
no